packages/office-ui-fabric-react/src/components/__snapshots__/DatePicker.Bounded.Example.tsx.shot

```diff
@@ -8,11 +8,35 @@ exports[`Component Examples renders DatePicker.Bounded.Example.tsx correctly 1`]
     When date boundaries are set (via minDate and maxDate props) the DatePicker will not allow out-of-bounds dates to be picked or entered. In this example, the allowed dates are 2018-5-13-2018-5-13
   </p>
   <div
-    className="ms-DatePicker"
+    className=
+        ms-DatePicker
+        {
+          box-shadow: none;
+          box-sizing: border-box;
+          margin-bottom: 0px;
+          margin-left: 0px;
+          margin-right: 0px;
+          margin-top: 0px;
+          padding-bottom: 0px;
+          padding-left: 0px;
+          padding-right: 0px;
+          padding-top: 0px;
+        }
   >
     <div>
       <div
-        className="ms-TextField"
+        className=
+            ms-TextField
+            ms-TextField
```

Comment:
That's odd, I wonder why `ms-TextField` appears twice?

Reply:
coming from TextField root, see my comment in DatePicker.styles.ts for the fix